### ISSUE TITLE
Unable to update 'purchaseOrderNumber' field when calling 'submitForSettlement'

### ISSUE THREAD

**Issue by vstudenichnik-insoft** [braintree/braintree_node#168]
### General information

* SDK/Library version: 2.23.0
* Environment: Both
* Language, language version, and OS: Node 4.9.1 on Ubuntu 18.04 LTS

### Issue description

According to the [documentation](https://developers.braintreepayments.com/reference/request/transaction/submit-for-settlement/node) for method `submitForSettlement`, it is possible to send additional parameters, including `purchaseOrderNumber`. But this does not happen, and the value of field `purchaseOrderNumber` remains the same as when calling method `sale`, which I do before that. Looking at the code, I noticed that only options `amount`, `orderId`, `descriptor` are taken into account.

```
  submitForSettlement(transactionId, amount, options) {
    let invalidKeysError;

    options = options || {};

    invalidKeysError = Util.verifyKeys(this._submitForSettlementSignature(), options);

    if (invalidKeysError) {
      return Promise.reject(invalidKeysError, null);
    }

    return this.gateway.http.put(`${this.config.baseMerchantPath()}/transactions/${transactionId}/submit_for_settlement`, {
      transaction: {amount, orderId: options.orderId, descriptor: options.descriptor} // not all options are passed
    }).then(this.responseHandler());
  }
```

So, do I understand correctly that this is a bug? Or is this the expected behavior?

Thanks in advance!

**Comment by billwerges** [comment]
Hi @vstudenichnik-insoft,

This does look like a bug on our end. We'll update here once we a fix is released.

**Comment by hollabaq86** [comment]
👋  @vstudenichnik-insoft sorry for the wait, this has been fixed and released in [version 3.3.0](https://github.com/braintree/braintree_node/blob/master/CHANGELOG.md#330)!
